web/access_control.go

```diff
@@ -78,6 +78,54 @@ func (s *AutograderService) isEnrolled(userID, courseID uint64) bool {
 	})
 }
 
+// isValidSubmission returns true if submitter has active course enrollment or
+// if submitting group belongs to the given course
+func (s *AutograderService) isValidSubmissionRequest(submission *pb.SubmissionRequest) bool {
+	if !submission.IsValid() {
+		fmt.Println("isValidSubmissionRequest failed: submission request not valid: ", submission)
```

Comment:
We should use the logger for debugging statements.

Reply:
Fixed.